Source/Public/Invoke-DockerTests.ps1

```diff
@@ -32,16 +32,24 @@ function Invoke-DockerTests {
     if (!(Test-Path $absoluteTestReportDir -PathType Container)) {
         New-Item $absoluteTestReportDir -ItemType Directory -Force | Out-Null
     }
+    $osType = Find-DockerOSType
+    $dockerSocket = Find-DockerSocket -OsType $osType
+    $configs = '/configs'
+    $report = '/report'
+    if ($osType -eq 'windows') {
+        $configs = 'C:/configs'
+        $report = 'C:/report'
+    }
```

Comment:
Removing the two lines before the if statement and then:

```suggestion
    } else {
	    $configs = '/configs'
	    $report = '/report'
	}
```

Reply:
Done